src/web-ui/src/partials/TermsAndConditions/TermsAndConditions.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="terms-and-conditions">
+    <p class="mb-1">
+      The Retail Demo Store is a demo application. All content displayed in the Retail Demo Store is fictitious and for
+      demonstration purposes only.
+    </p>
+
+    <p class="mb-1">
+      All images shown are from <a href="https://unsplash.com" target="_blank" rel="noopener noreferrer">Unsplash</a>.
```

Comment:
This may need to be modified (or linked to view with all CC and open source declarations) as additional fictitious products are added in the future. Good for now, though.